Lib/fontTools/feaLib/testdata/size.fea

```diff
@@ -0,0 +1,3 @@
+feature size {
+  parameters 10.0 0 0 0;
```

Comment:
with makeotf, `parameters 10.0` etc. compiles to ```<DesignSize value="10.0"/>``` etc.
The makeotf output is the same for  `parameters 100` (i.e. using "decipoints" notation).

With your patch, feaLib compiles the same `parameters 10.0` to `<DesignSize value="100.0"/>` (cf. "size.ttx" file); and it compiles `parameters 100` to `<DesignSize value="100"/>`.

See https://github.com/behdad/fonttools/pull/550/files#r56748809